test/azure/legacy/Expected/AcceptanceTests/Lro/lro/__init__.py

```diff
@@ -10,10 +10,17 @@
 from ._version import VERSION
 
 __version__ = VERSION
+
+try:
+    from ._patch import __all__ as _patch_all
+    from ._patch import *  # pylint: disable=unused-wildcard-import
+except ImportError:
+    _patch_all = []
+from ._patch import patch_sdk as _patch_sdk
+
 __all__ = ["AutoRestLongRunningOperationTestService"]
+__all__.extend([p for p in _patch_all if p not in __all__])
 
 # `._patch.py` is used for handwritten extensions to the generated code
 # Example: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/customize_code/how-to-patch-sdk-code.md
-from ._patch import patch_sdk
-
-patch_sdk()
+_patch_sdk()
```

Comment:
I think we could move the annotation to function annotation of `patch_sdk`, which will be more convenient for customers to find the reference link.
![image](https://user-images.githubusercontent.com/70930885/157610522-725f3549-6ca6-479a-8b4d-3035f9e33806.png)

Reply:
@msyyc I actually disagree there, we don't expect users to look into `__init__`, all they're supposed to update is `_patch.py`, so I'll move all documentation for `patch_sdk` to `_patch.py` so users can see it, otherwise they most likely won't

Reply:
Yeah, I totally agree with you that we shall not let users look into `__init__`. But for this case, the annotation in `__init__` is not moved to `_patch.py`. Is it because the code is not updated yet or by design or other thins?